components/adblock_rust_ffi/src/wrapper.h

```diff
@@ -56,6 +56,7 @@ typedef ADBLOCK_EXPORT struct FilterListMetadata {
 
   absl::optional<std::string> homepage;
   absl::optional<std::string> title;
+  uint16_t expires;
```

Comment:
just wonder, why 2 bytes integer is used here? Why not something like int64 which size matches to a word?

Reply:
Also, could we change name to `expires_in_hours`?

Reply:
Maximum valid value is 14 days i.e. 336 hours, so it will definitely always fit in a `uint16_t`.

Reply:
The names here correspond directly to the in-list representation, but I could add a doc comment at least

Reply:
added doc comment

Reply:
>Maximum valid value is 14 days i.e. 336 hours, so it will definitely always fit in a uint16_t.
It's definitely fit, but all integers with size less than a machine word are slower that (u)int64.

So using uint16 instead uint64 is about saving a few bytes against perf loss.
Smaller integer make sense for large vectors, serialization, etc.